spec/spec_helper.rb

```diff
@@ -1,2 +1,3 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'omniauth-multi-provider'
+require 'omniauth'
```

Comment:
Should this be required here? Maybe `multi_provider.rb` should require `omniauth` since it references `OmniAuth.config` directly?